test/e2e/psp.feature.ts

```diff
@@ -40,19 +43,20 @@ const readBeaconValue = async (airnodeAddress: string, templateId: string, dapiS
   const beaconId = hre.ethers.utils.keccak256(
     hre.ethers.utils.solidityPack(['address', 'bytes32'], [airnodeAddress, templateId])
   );
-  const dapiServerResponse = await dapiServer.connect(voidSigner).readWithDataPointId(beaconId);
 
-  return dapiServerResponse[0].toNumber();
+  try {
```

Comment:
I'm a bit confused here. AFAIK the contract method can return an uint224 therefore it seems kind of risky that we do a `toNumber()` with a try-catch. Why do we even need to cast it to javascript number? Can we just keep using the ethers.BigNumber returned by the contract or maybe just cast it to string?

Reply:
You make a good point, we can keep it as **BigNumber**